source/slang/slang-ir.h

```diff
@@ -1190,6 +1190,25 @@ struct IRStructType : IRType
     IR_LEAF_ISA(StructType)
 };
 
+struct IRAssociatedType : IRType
+{
+    IR_LEAF_ISA(AssociatedType)
+};
+
+struct IRInterfaceRequirementEntry : IRInst
+{
+    // The AST-level requirement
+    IRUse requirementKey;
+
+    // The IR-level value that represents the declaration of the requirement
+    IRUse requirementVal;
```

Comment:
It isn't any kind of documented policy, but I've largely moved away from bothering to declare the `IRUse` members in derived `IRInst` types (even if that makes their memory layout unclear to somebody reading the code).

We probably need to establish an official policy on this and then make all the `IRInst`-derived types follow that policy.

Reply:
Removed.